doc/user/content/ops/optimization.md

```diff
@@ -20,26 +20,71 @@ Building an efficient index for distinct **clauses** and **operators** can be pu
+view, or table. `$1`, `$2`, etc. mean literals, e.g., `42`, or `'foo'`.)
 
-Clause                    | Index                                                                   |
---------------------------|-------------------------------------------------------------------------|
-`WHERE x = $1`            | `CREATE INDEX ON view_name (x);`                                        |
-`WHERE x IN ($1)`         | `CREATE INDEX ON view_name (x);`                                        |
-`WHERE x * 2 = $1`        | `CREATE INDEX ON view_name (x * 2);`                                    |
-`WHERE upper(x) = $1`     | `CREATE INDEX ON view_name (upper(x));`                                 |
-`WHERE x = $1 AND y = $2` | `CREATE INDEX ON view_name (x, y);`                                     |
-`WHERE x = $1 OR y = $2`  | `CREATE INDEX ON view_name (x);`<br /> `CREATE INDEX ON view_name (y);` |
+Clause                    | Index                                |
+--------------------------|--------------------------------------|
+`WHERE x = $1`            | `CREATE INDEX ON collection (x);`    |
+`WHERE x IN ($1, $2, $3)` | `CREATE INDEX ON collection (x);`        |
+`WHERE (x, y) IN (($1, $2), ($3, $4))` | `CREATE INDEX ON collection (x, y);`     |
+`WHERE x = $1 AND y = $2` | `CREATE INDEX ON collection (x, y);`     |
+`WHERE (x = $1 AND y = $2) OR (x = $3 AND y = $4`) | `CREATE INDEX ON collection (x, y);`     |
+`WHERE x * 2 = $1`        | `CREATE INDEX ON collection (x * 2);`    |
+`WHERE upper(x) = $1`     | `CREATE INDEX ON collection (upper(x));` |
 
-**Note:** to speed up a query using a multi-column index, as in `WHERE x = $1 AND y = $2`, the query must use all the fields in the index chained together via the `AND` operator.
+You can verify that the index is being used for a lookup with `EXPLAIN`, which should show `lookup`:
+```
+materialize=> EXPLAIN SELECT * FROM foo WHERE x = 6 AND y = 8;
+                            Optimized Plan
+----------------------------------------------------------------------
+ Explained Query (fast path):                                        +
+   Project (#0, #1)                                                  +
+     ReadExistingIndex materialize.public.foo_x_y lookup value (6, 8)+
+                                                                     +
+ Used Indexes:                                                       +
+   - materialize.public.foo_x_y                                      +
+```
+
+**Matching multi-column indexes to multi-column `WHERE` clauses:** In general, your index key should exactly match the columns that are constrained in your `WHERE` clause. In more detail:
+- **Attention:** If your `WHERE` clause constrains fewer fields than your index key includes, then **the index will not be used**. For example, an index on `(x, y)` cannot be used to speed up `WHERE x = $1`.
```

Comment:
This section is very good cause it encapsulates the key details while using queries and understanding Materialize behavior. Still, I'm unsure how to display the text structure, but we can leave it for the future.